### PR TITLE
Edit message dialog

### PR DESCRIPTION
**Pull request by BeatC**

### CODE REVIEW

File: app/features/task/components/create_task_dialog.js [hunk 1]
```diff
@@ -53,31 +24,54 @@ class CreateTaskDialog extends React.Component {
...
-      <Dialog modal={true}
-              title="Create Task"
-              actions={this.actions}
-              open={this.props.open}
-              contentStyle={styles}>
-        <TaskForm />
-      </Dialog>
-    );
+    let form = <TaskForm />;
+    return this.getDialog(form);
```
Comment (DmitriyVelychko) [inline app/features/task/components/create_task_dialog.js]: This is looks strange

File: app/features/task/components/edit_task_dialog.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      time,
+      repeatEveryDay,
+      repeatEveryWeek
+    });
+  }
+
+  _onDelete() {
+    let {
+      onDelete,
+      task
```
Comment (DmitriyVelychko) [inline app/features/task/components/edit_task_dialog.js]: currentTask

Comment (BeatC) [inline app/features/task/components/edit_task_dialog.js]: I'll update that.

File: app/features/task/components/create_task_dialog.js (hunk 1 above)
Comment (BeatC) [inline app/features/task/components/create_task_dialog.js]: Why? :)

**Pull request merged**